os/include/net/lwip/lwipopts.h

```diff
@@ -63,9 +63,11 @@
  * In this platform, DHCP client and server are supported through netutils
  * Instead of using lwIP DHCP, please use netutils dhcpc and dhcpd
  */
+#define ENABLE_LOOPBACK                 1
 #define LWIP_DHCPS                      0
 #define LWIP_DHCP                       0
 #define LWIP_NETIF_API                  1
+#define LWIP_ETHERNET                   1
```

Comment:
Please use CONFIG_* checks and assignments to assign 0 or 1 to LWIP_ETHERNET, if this flag is to be enabled only for qemu. Similar comment applied to ENABLE_LOOPBACK.